taqueria-vscode-extension-web-ui/src/data-editors/BoolEditor.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { isObject } from '../Helpers';
+
+export const BoolEditor = (
+	{ value, onChange }: { value: any; onChange: (value: { prim: 'True' | 'False' }) => void },
+) => {
```

Comment:
```suggestion
export const BoolEditor: React.FC<{
  value: any;
  onChange: (value: { prim: "True" | "False" }) => void;
}> = ({ value, onChange }) => {
```

Reply:
Can you give more context? Or a link? I don't know what that is. Thanks

Reply:
Looks like I'm outdated 😅 
https://github.com/facebook/create-react-app/pull/8177

Reply:
But we can at least have a separate type for props shape, above the component function definition.